pages/common/virt-clone.md

```diff
@@ -0,0 +1,8 @@
+# virt-clone
+
+> Clone a libvirt virtual machine.
+> More information: <https://virt-manager.org/>.
```

Comment:
This link is really not a good one, because the site is very lacking in accessible information, as can be seen [here](https://github.com/tldr-pages/tldr/issues/5197#issue-795578123).  The man page, on the other hand, does provide a lot of nice detail.
```suggestion
> More information: <https://manned.org/virt-clone>.
```
(As we've discussed before, using the man page for everything is not the best idea, but in this case I do believe it's an improvement over this current link.)

Reply:
I've made this change in my latest commit. 

This was the link used in the virt-install tldr-page. I'll stop assuming that the More Information links in existing pages are necessarily correct.